ui/revenuecatui/src/main/kotlin/com/revenuecat/purchases/ui/revenuecatui/customercenter/views/AccountDetailsSection.kt

```diff
@@ -0,0 +1,295 @@
+                    vertical = 12.dp,
+                ),
+            verticalArrangement = Arrangement.spacedBy(CustomerCenterConstants.Layout.ITEMS_SPACING),
+        ) {
+            originalPurchaseDate?.let { purchaseDate ->
+                PurchaseDateSection(
+                    purchaseDate = purchaseDate,
+                    localization = localization,
+                    textColor = textColor,
+                    secondaryColor = secondaryColor,
+                    showSpacer = appUserId.isNotBlank(),
+                )
+            }
+
+            if (appUserId.isNotBlank()) {
+                UserIdSection(
+                    appUserId = appUserId,
+                    localization = localization,
+                    textColor = textColor,
+                    secondaryColor = secondaryColor,
+                )
+            }
+        }
+    }
+}
+
+@Composable
+private fun PurchaseDateSection(
+    purchaseDate: String,
+    localization: CustomerCenterConfigData.Localization,
+    textColor: Color,
+    secondaryColor: Color,
+    showSpacer: Boolean,
+) {
+    Column {
+        Text(
+            text = localization.commonLocalizedString(
+                CustomerCenterConfigData.Localization.CommonLocalizedString.DATE_WHEN_APP_WAS_PURCHASED,
+            ),
+            style = MaterialTheme.typography.labelLarge,
+            color = secondaryColor,
+        )
+
+        Text(
+            text = purchaseDate,
+            style = MaterialTheme.typography.bodyMedium,
+            color = textColor,
+        )
+
+        if (showSpacer) {
+            Spacer(modifier = Modifier.height(8.dp))
+        }
+    }
+}
+
+@Composable
+private fun UserIdSection(
+    appUserId: String,
+    localization: CustomerCenterConfigData.Localization,
+    textColor: Color,
+    secondaryColor: Color,
+) {
+    val clipboard = LocalClipboardManager.current
+
+    Row(
+        modifier = Modifier.fillMaxWidth(),
+        verticalAlignment = Alignment.CenterVertically,
+    ) {
+        Column(
+            modifier = Modifier.weight(1f),
+        ) {
+            Text(
+                text = localization.commonLocalizedString(
+                    CustomerCenterConfigData.Localization.CommonLocalizedString.USER_ID,
+                ),
+                style = MaterialTheme.typography.labelLarge,
+                color = secondaryColor,
+            )
+
+            SelectionContainer {
+                Text(
+                    text = appUserId,
+                    style = MaterialTheme.typography.bodyMedium.copy(fontFamily = FontFamily.Monospace),
+                    color = textColor,
+                )
+            }
+        }
+
+        IconButton(
+            onClick = {
+                clipboard.setText(AnnotatedString(appUserId))
+            },
+        ) {
+            Icon(
+                imageVector = Icons.Outlined.ContentCopy,
+                contentDescription = "Copy",
```

Comment:
Hmm I think `contentDescription` here should be either `null` or be localized... I think until now, for icons like these, we've mostly left it as null but I guess we don't have any accompanying string here for that, so I think the proper thing to do here would be to localize it... Maybe we can do it in a separate PR though.